xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -268,6 +268,114 @@ std::string CGUIWindowVideoNav::GetQuickpathName(const std::string& strPath) con
   }
 }
 
+SELECT_FIRST_UNWATCHED_ITEM CGUIWindowVideoNav::SelectFirstUnwatchedItem()
```

Comment:
I had it that way originally, but it conflicted with the name of the method, SelectFirstUnwatchedItem, and Visual Studio didn't like it when trying to cast the int received from CSettings to the enum.